pkg/ovs/ovn-nb-logical_switch_port.go

```diff
@@ -25,6 +25,21 @@ func (c *OVNNbClient) CreateLogicalSwitchPort(lsName, lspName, ip, mac, podName,
 
 	// ignore
 	if exist {
+		if portSecurity && len(securityGroups) != 0 {
+			lsp, err := c.GetLogicalSwitchPort(lspName, true)
+			if err != nil {
+				klog.Error(err)
+			} else {
+				sgList := strings.Split(securityGroups, ",")
+				if _, err := c.SetLogicalSwitchPortSecurityGroup(lsp, "add", sgList...); err != nil {
+					klog.Errorf("set logical switch port %s security groups %s: %v", lsp.Name, securityGroups, err)
```

Comment:
return err

Reply:
The code has been updated to another style.